src/saveload/afterload.cpp

```diff
@@ -3125,6 +3125,13 @@ bool AfterLoadGame()
 		}
 	}
 
+	/* Make sure all industries have control flags cleared */
+	if (IsSavegameVersionBefore(SLV_GS_INDUSTRY_CONTROL)) {
+		for (Industry *i : Industry::Iterate()) {
+			i->ctlflags = INDCTL_NONE;
+		}
+	}
```

Comment:
I'm reasonably certain this is unnecessary, as the object (prior to being filled with saveload) will be initialised with INDCTL_NONE anyway